services/apps/alcs/src/providers/typeorm/migrations/1685382185264-unlink_journal.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class unlinkJournal1685382185264 implements MigrationInterface {
+  name = 'unlinkJournal1685382185264';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."application_staff_journal" DROP CONSTRAINT "FK_a074329b9f848c30b4165a79889"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."application_staff_journal" RENAME TO "staff_journal";`,
+    );
+
+    await queryRunner.query(
```

Comment:
The only concern that I have is probably the fact that there is no FK on db for staff journal and no way to easily tell based on the record that it belongs to application or NOI

Reply:
Yeah I was thinking the same thing, should I add a staff journal column to Application and NOI to keep an FK? The good news is that we never actually delete NOIs or Applications.